project-base/storefront/components/Pages/StoreDetail/StoreDetailContent.tsx

```
@@ -25,100 +25,93 @@ export const StoreDetailContent: FC<StoreDetailContentProps> = ({ store }) => {
-                            {store.street}
-                            <br />
-                            {store.city}
-                            <br />
-                            {store.postcode}
-                            <br />
-                            {store.country.name}
-                        </p>
-                    </InfoItem>
+                        {!!store.description && (
+                            <StoreSection title={t('Store description')}>
+                                <p dangerouslySetInnerHTML={{ __html: store.description }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)